utils/cargo-tfhe-lints/src/main.rs

```diff
@@ -0,0 +1,38 @@
+use std::process::{exit, Command};
+
+fn get_supported_rustc_version() -> &'static str {
+    const TOOLCHAIN_FILE: &str = include_str!("../rust-toolchain-link");
+
+    TOOLCHAIN_FILE
+        .lines()
+        .find(|line| line.starts_with("channel ") || line.starts_with("channel="))
+        .and_then(|line| {
+            line.rsplit('=')
+                .next()
+                .unwrap()
+                .trim()
+                .strip_prefix('"')
+                .unwrap()
+                .strip_suffix('"')
+        })
+        .unwrap()
+}
+
+fn main() {
+    let cargo_args = std::env::args().skip(2).collect::<Vec<_>>();
+    let toolchain = format!("+{}", get_supported_rustc_version());
+
```

Comment:
let toolchain = toolchain.as_str();

avoids the clone line 26 I think

Reply:
done